src/Faker/Generator.php

```diff
@@ -11,6 +11,8 @@
  * @property string $title
  * @property string $titleMale
  * @property string $titleFemale
+ * @property string $secondPersonalNameMale
```

Comment:
no, this is only for generally-available formatters, not for locale-specific ones.